resources/css/bem/daily-challenge.less

```diff
@@ -5,9 +5,28 @@
   background: hsl(var(--hsl-b4));
   border-radius: @border-radius-large;
   min-width: 0;
+  position: relative;
   display: flex;
   align-items: center;
-  padding: 3px;
+  padding: 1px;
+  border: 2px solid transparent;
+
+  &--played-today {
+    border-color: @osu-colour-lime-1;
+
+    &::before {
+      .fas();
+      background-color: @osu-colour-b6;
+      border-radius: 50%;
```

Comment:
while at it, this doesn't actually do anything, does this? The fa icon is already rounded and all

Reply:
The fontawesome icon is rounded but it has a hollow interior where the tick is. So without some hackage the border will peek out of the hollow interior, thus this + the `background-color` + `border-radius` specs are supposed to cover up anything that's in the background (particularly the border).

Reply:
oh. So that also explained the weirdly subtle black outline... 🤔 

I suppose an alternative would be to use normal check mark coupled with circle background, centered using `.center-content()`

Reply:
Yeah that's a fair shout. Adjusted further in https://github.com/ppy/osu-web/pull/11780/commits/bf0333ab6ad91e4aa4b16b9c9d66c6a92c65697b.

I also took the opportunity to resize the tick itself because for whatever reason when using the pre-baked fontawesome `check-circle` the check was much larger than it actually was in the figma. It should be there-or-thereabouts matching now.